src/ReactTouchHandler.js

```diff
@@ -95,10 +95,6 @@ class ReactTouchHandler {
   }
 
   onTouchStart(/*object*/ event) {
-    if (this._preventDefault) {
```

Comment:
From https://github.com/schrodinger/fixed-data-table-2/pull/679, where we don't prevent default the touch handlers for `onTouchStart` and `onTouchEnd` events.

This is based on the suggestion from MDN docs (https://developer.mozilla.org/en-US/docs/Web/API/Touch_events#handling_clicks).

> Since calling preventDefault() on a [touchstart](https://developer.mozilla.org/en-US/docs/Web/API/Element/touchstart_event) or the first [touchmove](https://developer.mozilla.org/en-US/docs/Web/API/Element/touchmove_event) event of a series prevents the corresponding mouse events from firing, it's common to call preventDefault() on touchmove rather than touchstart. That way, mouse events can still fire and things like links will continue to work. Alternatively, some frameworks have taken to re-firing touch events as mouse events for this same purpose.